DataFormats/L1TParticleFlow/src/HPSPFTau.cc

```diff
@@ -34,57 +32,53 @@ ostream& operator<<(ostream& os, const l1t::HPSPFTau& l1PFTau) {
   os << "seed:";
   if (l1PFTau.isChargedPFCandSeeded()) {
     os << " chargedPFCand";
-  } else if (l1PFTau.isPFJetSeeded()) {
+  } else if (l1PFTau.isJetSeeded()) {
     os << " PFJet";
-  } else {
-    cms::Exception ex("InconsistentTau");
-    ex.addContext("Calling HPSPFTau::operator <<");
-    ex.addAdditionalInfo("This tau is not seed by either a chargedPFCand or a PFJet!");
-    throw ex;
-  }
+  } else
+    assert(0);
   os << std::endl;
   os << "signalPFCands:" << std::endl;
-  for (const auto& l1PFCand : l1PFTau.signalAllL1PFCandidates()) {
+  for (auto l1PFCand : l1PFTau.signalAllL1PFCandidates()) {
```

Comment:
here and below, please add back the `const&`

Reply:
Done